code/__DEFINES/mobs.dm

```diff
@@ -60,9 +60,17 @@
 #define DEFAULT_BODYPART_ICON_ORGANIC 'icons/mob/human_parts_greyscale.dmi'
 #define DEFAULT_BODYPART_ICON_ROBOTIC 'icons/mob/augmentation/augments.dmi'
 
+
 #define MONKEY_BODYPART "monkey"
-#define ALIEN_BODYPART "alien"
-#define LARVA_BODYPART "larva"
+#define ALIEN_BODYPART 	"alien"
+#define LARVA_BODYPART 	"larva"
+
+
+#define BODY_DEFAULT 0
```

Comment:
Misleading define, 0 is not the default body

Reply:
I can rename it to null_body, that's fine, or just remove it since it's not being used either way.  I just wanted a null option that didn't accept any body parts

Reply:
> 
> 
> can you talk me through the part origin vs animal origin bit? Why arent we just using animal origin here?

The main reason I didn't just use animal origin for everything is that animal_origin needs to be a string for it to be parsed in carbon_update_icons.dm I tried just using the bitmap and it made monkeys (and maybe other mobs, idk) invisible.

Reply:
> > 
> > 
> > can you talk me through the part origin vs animal origin bit? Why arent we just using animal origin here?
> 
> The main reason I didn't just use animal origin for everything is that animal_origin needs to be a string for it to be parsed in carbon_update_icons.dm I tried just using the bitmap and it made monkeys (and maybe other mobs, idk) invisible.

mentioning @ExcessiveUseOfCobblestone here because i dont think he noticed you replied to him down in a review

Reply:
changed to NULL_BODY